.github/workflows/pull_request_labeler.yml

```
@@ -0,0 +1,12 @@
+name: "Pull Request Labeler"
```

```suggestion
name: "Pull request labeler"
```

Just to match our existing capitalisation in other files.